tests/test_core.py

```diff
@@ -30,7 +31,13 @@ def test_filelog():
     assert log._get_now() == ""
     assert log.title("Test") == "Test"
     log.clear()
-    assert log.get_size() == 44
```

Comment:
The intention of this line was to cover the get_size() method in the filelog. Would it be possible to cover it without breaking the CI?

Reply:
Aha, sure, will do it. We shouldn't have regression in coverage

Reply:
@masqu3rad3 FYI the log file size changes depending on the platform with `log.get_size()` function.  In the CI, job the get_size was returning 40 instead of 44.  My guess for the reason was the 2 character newline on windows vs 1 character newline on linux (CI runner is linux).

Reply:
Yes I noticed that. We can just test it if the value is integer or not maybe. We don't need to test the exact value.

Reply:
I ended up doing a platform specific test and adding a windows CI runner :D  I am not sure if this is the way to go, but we can merge it anyways.

I am curious if we explicitly set the encoding for the logging file handler if the difference in size on different platforms would disappear or not.  Maybe I can test it some other time